src/View/Home/rules.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheet %}
+	<link rel="stylesheet" href="/assets/css/rules.css">
+{% endblock %}
+
+{% block content %}
+	<h1 class="rule-title">Bienvenue dans Happiness & Glitters</h1>
+	<h2 class="rule-title">Comment jouer ? (et peut-être gagner)</h2>
+	<div class="rule">
```

Comment:
since you have a list of rules, please wrap all your rules inside a ul tag and change the DIV tag in LI TAG for each Rule